static/app/views/replays/detail/breadcrumbs/breadcrumbFilters.tsx

```diff
@@ -0,0 +1,43 @@
+import {CompactSelect} from 'sentry/components/compactSelect';
+import SearchBar from 'sentry/components/searchBar';
+import {t} from 'sentry/locale';
+import useBreadcrumbFilters from 'sentry/views/replays/detail/breadcrumbs/useBreadcrumbFilters';
+import FiltersGrid from 'sentry/views/replays/detail/filtersGrid';
+
+type Props = {
+  frames: undefined | unknown[];
+} & ReturnType<typeof useBreadcrumbFilters>;
+
+function BreadcrumbFilters({
+  frames,
+  getBreadcrumbTypes,
+  searchTerm,
+  setSearchTerm,
+  setType,
+  type,
+}: Props) {
+  const breadcrumbTypes = getBreadcrumbTypes();
+  return (
+    <FiltersGrid>
+      <CompactSelect
+        triggerProps={{prefix: t('Type')}}
+        triggerLabel={type.length === 0 ? t('Any') : null}
+        multiple
+        options={breadcrumbTypes}
+        size="sm"
+        onChange={selected => setType(selected.map(_ => _.value))}
```

Comment:
Generally, we use `_` to signify to the linter that it is unused and can be ignored.

e.g.

```
function foo(a, _b, c) {
// _b will be ignored by linter if it's unused
}
```

I prefer to destructure in the callback to save yourself from coming up with an argument name.

```suggestion
        onChange={selected => setType(selected.map(({value}) => value))}
```